media/js/base/mozilla-monitor-button.js

```diff
@@ -22,15 +22,32 @@ Mozilla.MonitorButton = (function() {
     var buttonURL = monitorButton.getAttribute('href');
     var destURL = monitorButton.getAttribute('data-action') + 'metrics-flow';
 
+    // collect values from monitor button
+    var utmSource = buttonURL.match(/(utm_source=.*?)&/)[1].toString();
```

Comment:
We have a [small helper library](https://github.com/mozilla/bedrock/blob/master/media/js/base/search-params.js#L30-L31) in the base bundle that makes this kind of task much easier to do without needing custom code. Might that make this easier to avoid needing to keep the params in a certain order?

Reply:
Having similar issue with this function because the query is being split by `&`'s

https://github.com/mozilla/bedrock/blob/2ff934c938ca23ba380999655ec8ad2165132b18/media/js/base/search-params.js#L34

so the `utm_source` key in this object is the remainder of the url because it begins with `?`. :( 
<img width="303" alt="Screen Shot 2019-08-09 at 2 05 29 PM" src="https://user-images.githubusercontent.com/30483988/62809071-c7c28280-baae-11e9-9f55-3f0e1713e72b.png">

Going to try and tweak this function to account for this.